Makefile

```diff
@@ -103,7 +106,7 @@ migration-revert:
 migration-status:
 	docker-compose run --rm --no-deps app vendor/doctrine/migrations/bin/doctrine-migrations migrations:status --configuration=vendor/wmde/fundraising-store/migrations.yml
 
-ci: covers phpunit cs npm-ci validate-app-config validate-campaign-config stan
+ci: covers phpunit-with-coverage cs npm-ci validate-app-config validate-campaign-config stan
```

Comment:
I often run `make ci` before committing, so I'd like us to have a different CI command without coverage. Maybe `ci` and `ci-with-coverage`?